src/par_iter/mod.rs

```diff
@@ -459,6 +459,14 @@ pub trait ParallelIterator: Sized {
         reduce(self, MUL)
     }
 
+    /// DEPRECATED. Use .product() instead.
+    #[deprecated]
```

Comment:
should be something like this:

```rust
#[deprecated(since = "v0.6.0", note = "name changed to `product()` to match sequential iterators)]
```

I put 0.6.0 because I think the next release includes a few breaking changes...but I can't remember for sure...